IdentityCore/xcconfig/identitycore__common__ios.xcconfig

```diff
@@ -1,6 +1,7 @@
-SDKROOT = iphoneos;
-CODE_SIGN_IDENTITY = "iPhone Developer";
-IPHONEOS_DEPLOYMENT_TARGET = 9.3;
-LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
-TARGETED_DEVICE_FAMILY = 1,2;
-ENABLE_BITCODE = YES;
+SDKROOT = iphoneos
+CODE_SIGN_IDENTITY = "iPhone Developer"
+IPHONEOS_DEPLOYMENT_TARGET = 9.3
```

Comment:
nit: should we have it 9.0 for clarity?

Reply:
There are API differences between 9.0 and 9.3, we only support the latest patched version on any major version of iOS.